README.md

```diff
@@ -83,7 +83,7 @@ See this [blog post](https://rossta.net/blog/how-to-specify-local-ruby-gems-in-y
 * nginx or another reverse proxy
 
 ## Funding
-The [Medieninnovationszentrum Babelsberg](http://miz-babelsberg.de) supports the development of additional modules.
+The [Medieninnovationszentrum Babelsberg](http://miz-babelsberg.de) supported the development of additional 'Crowdlupe' modules.
```

Comment:
The mentioning of `Crowdlupe` won't be very helpful as the word i's nowhere to be found in the entire code base, therefore I'll delete that one.